imagefeedview/src/test/java/com/oliveroneill/imagefeedview/adapter/PhotoPagerAdapterTest.kt

```diff
@@ -18,30 +14,23 @@ class PhotoPagerAdapterTest {
     lateinit var adapter : PhotoPagerAdapter<String>
     lateinit var mockController : ImageFeedController<String>
     lateinit var mockView : PhotoPagerAdapter.ViewHolder
-    lateinit var mockImageView : GestureImageView
     @Before
     fun setup() {
-        mockController = mock()
+        mockController = mockk(relaxed = true)
```

Comment:
here you can see that all the chains are mocked automatically

Reply:
That's a really nice feature!